fuzz/tls_init/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+project(fuzz_init CXX)
+
+set(CMAKE_C_FLAGS ${CMAKE_C_FLAGS})
+set(CMAKE_CXX_COMPILER "/usr/bin/clang++")
+set(CMAKE_CXX_FLAGS "-fsanitize=address,fuzzer -g -fPIE ${CMAKE_CXX_FLAGS}")
+
+set(INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/../../src/include
+                    ${CMAKE_CURRENT_SOURCE_DIR}/../../src/include/edl
```

Comment:
edl is for SGX mode. There is no need to include edl dir in host mode. Please recheck this.